static/app/utils/useProjectKeys.tsx

```diff
@@ -0,0 +1,47 @@
+import {useEffect, useState} from 'react';
+
+import {Organization, Project, ProjectKey, RequestState} from 'sentry/types';
+
+import useApi from './useApi';
+
+export function useProjectKeys({
+  organization,
+  project,
+}: {
+  organization: Organization | null;
+  project: Project | null;
+}) {
+  const api = useApi();
+  const [response, setResponse] = useState<RequestState<ProjectKey[] | null>>({
```

Comment:
Why is this `null` needed? Does it represent something different from the empty array?

Reply:
its not anymore; its from when i had it as `string | null` while i was picking out `data[0]?.dsn.public` 

changed since we moved to a common util hook

i've updated it now